CODEWALK.md

```diff
@@ -346,6 +346,118 @@ essentially who is storing what data, for what fee and which sectors have been s
 The keystore contains the binary encoded peer key for interacting securely over the network. 
 This data lives in a file at `$HOME/.filecoin/keystore/self`.
 
+## libp2p Networking
```

Comment:
Blocking: update the table of contents

Nit: lowercase 'N' for consistency with other titles. Or maybe `Networking (libp2p)`